codecademy-style.gemspec

```diff
@@ -18,9 +18,8 @@ Gem::Specification.new do |spec|
   spec.metadata['homepage_uri'] = spec.homepage
   spec.metadata['source_code_uri'] = spec.homepage
 
-  unless spec.respond_to?(:metadata)
-    raise 'RubyGems 2.0 or newer is required to protect against public gem pushes.'
-  end
+  error_msg = 'RubyGems 2.0 or newer is required to protect against public gem pushes.'
+  raise error_msg unless spec.respond_to?(:metadata)
```

Comment:
Lint fix for this error that I kept getting 
https://app.circleci.com/pipelines/github/codecademy-engineering/codecademy-ruby-style/38/workflows/345ad2e8-2446-4dfa-b525-c8b1f7d3d1af/jobs/33

Reply:
Weird that this is only coming up now. I'm fine with the change.